terraform/services/github-actions-runner/main.tf

```
@@ -108,7 +108,7 @@ module "github-actions-runner" {
 
   instance_target_capacity_type = "on-demand"
   instance_types = [
-    "t3.xlarge",
+    "m6i.xlarge",
```

Any concerns over selecting [a modestly less expensive](https://instances.vantage.sh/?filter=m6i.xlarge|m6a.xlarge&compare_on=true&selected=m6i.xlarge,m6a.xlarge) AMD variant over the Intel variant?
```suggestion
    "m6a.xlarge",
```

---

No concerns from me. Any differences between Intel and AMD we run into in these workflows would be pretty extreme edge cases.

---

@gsf Instance type has been updated to m6a.xlarge. Please approve if this looks good.